benchmark/alpa/benchmark_one_case_gpt_bert_inference.py

```diff
@@ -170,16 +171,24 @@ def benchmark_gpt_inference_internal(model_type,
 
     if profile_stage_execution_time:
         exec_info = executable.get_stage_execution_info()
+        timelines = list(zip(*exec_info))
+        # drop warmup case
+        timelines = timelines[1:]
         dump_chrome_tracing(
-            zip(*exec_info),
-            f"./chrome_trace/bs={benchmark_case.batch_size},pp={num_manual_pipeline_stages}.json"
+            timelines,
+            f"./chrome_trace/bs={benchmark_case.batch_size},op={benchmark_case.parallel_args.op},pp={benchmark_case.parallel_args.pp}.json"
         )
+        avg_stage_latencies = compute_avg_stage_latencies(timelines)
+        assert len(avg_stage_latencies) == num_manual_pipeline_stages
+    else:
+        avg_stage_latencies = None
 
     # Compute statistics
     tflops, parameter_count = compute_gpt_inference_statistics(
         benchmark_case, latencies, virtual_mesh.num_devices_per_host)
     metadata = {
         "latencies": latencies,
```

Comment:
Why do we need this metadata? I think `latencies` is already in the return value

Reply:
This `latencies` only contains end-to-end latency for each invocation.

Reply:
This latencies only contains end-to-end latency for each invocation

Reply:
Is it the same as the return value? https://github.com/alpa-projects/alpa/blob/c5ca9d63fab25956ec738f93cc48927cc25eab59/benchmark/alpa/benchmark_one_case_gpt_bert_inference.py#L211

Reply:
Is it the same as the return value here?  https://github.com/alpa-projects/alpa/blob/c5ca9d63fab25956ec738f93cc48927cc25eab59/benchmark/alpa/benchmark_one_case_gpt_bert_inference.py#L211

The script already prints the mean and std of the latencies. Why are the mean and latency not enough?

Reply:
Because we need fine-grained per-stage latency?

Reply:
No, the `latencies` is a list of end-to-end latency for each invocation. Now I removed the per-stage latency from metadata and only log it inside the benchmark_one_case_gpt_bert_inference if `profile_stage_execution_time` is set to True.

Reply:
I meant `metadata` is necessary but L210 is not necessary. Because the variable `latency` in L210 is already in the return value.

Reply:
I see. I'll remove it.